src/app/users/users-routing.module.ts

```diff
@@ -4,90 +4,98 @@ import { IsAuth } from "../core/_guards/auth.guard";
               permission: 'User'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
           path: ':id/edit', component: EditUsersComponent,
           data: {
               kind: 'edit',
               breadcrumb: 'Edit User',
               permission: 'User'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
           path: 'all-users', component: AllUsersComponent,
           data: {
               kind: 'all-users',
               breadcrumb: 'All Users',
               permission: 'User'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
           path: 'global-permissions-groups', component: GlobalpermissionsgroupsComponent,
           data: {
               kind: 'globalpermissionsgp',
               breadcrumb: 'Global Permissions Groups',
               permission: 'RightGroup'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
-          path: 'global-permissions-groups/new', component: NewGlobalpermissionsgroupsComponent,
+          path: 'global-permissions-groups/new', component: FormComponent,
           data: {
-              kind: 'new-globalpermissionsgp',
+              kind: 'newglobalpermissionsgp',
+              type: 'create',
+              path: SERV.ACCESS_PERMISSIONS_GROUPS,
               breadcrumb: 'New Global Permissions Groups',
               permission: 'RightGroup'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
           path: 'global-permissions-groups/:id/edit', component: EditGlobalpermissionsgroupsComponent,
           data: {
               kind: 'edit-gpg',
               breadcrumb: 'Edit Global Permissions Group',
               permission: 'RightGroup'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
           path: 'access-groups', component: GroupsComponent,
           data: {
               kind: 'access-groups',
               breadcrumb: 'Access Groups',
               permission: 'GroupAccess'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
-          path: 'access-groups/new', component: CUGroupComponent,
+          path: 'access-groups/new', component: FormComponent,
           data: {
-              kind: 'new-access-groups',
+              kind: 'newaccessgroups',
+              type: 'create',
+              path: SERV.ACCESS_GROUPS,
               breadcrumb: 'New Access Group',
               permission: 'GroupAccess'
           },
-          canActivate: [IsAuth,CheckPerm]},
+          canActivate: [CheckPerm]},
         {
-          path: 'access-groups/:id/edit', component: CUGroupComponent,
+          path: 'access-groups/:id/edit', component: FormComponent,
           data: {
```

Comment:
We should probably create an interface for router data

Reply:
Done